test/e2e/build_test.go

```diff
@@ -552,7 +554,7 @@ subdir**`
 		Expect(ioutil.WriteFile(filepath.Join(subdirPath, "extra"), contents.Bytes(), 0644)).
 			ToNot(HaveOccurred())
 		randomFile := filepath.Join(subdirPath, "randomFile")
-		dd := exec.Command("dd", "if=/dev/random", "of="+randomFile, "bs=1G", "count=1")
+		dd := exec.Command("dd", "if=/dev/urandom", "of="+randomFile, "bs=1G", "count=1")
```

Comment:
I can never remember which is the fast pseudo-random one?  We should use that.  The entropy-pool in these VMs is a problem we've struggled with in different contexts before.

Reply:
You can safely assume that if I changed it, with a corresponding acknowledgment in the commit message, it is the correct change. But you can also confirm at `random(4)`:
>When read, the /dev/urandom device returns random bytes using a pseudo-random  number generator seeded from the entropy pool.  Reads from this device do not block (i.e., the CPU is not yielded), but  can  incur  an appreciable delay when requesting large amounts of data.

I'm guessing that we're hitting this delay, I just don't know what to do about it, so I'm leaving that as SEP (#15014).

Reply:
In case anybody does some archaeology on this change:  GCE uses QEMU/KVM VMs and does not provide a virtual RNG for them (we asked).  That means the entropy pool is incredibly limited, and we rely on the (pre-enabled) `rngd` service to refill it.  In other words, tests should never rely on fast good random numbers, only fast less-good ones or VERY slow good ones.